src/metpy/xarray.py

```diff
@@ -162,24 +167,41 @@ def _fixup_coordinate_map(self, coord_map):
             if coord_map[axis] is not None and not isinstance(coord_map[axis], xr.DataArray):
                 coord_map[axis] = self._data_array[coord_map[axis]]
 
+        # Adapt to old CF axis dict keys
+        # TODO: Remove in v1.0
+        if any(cf_axis in coord_map for cf_axis in cf_to_metpy_axes):
+            warnings.warn('The use of CF axes types in the coordinate mapping has been '
```

Comment:
To clarify, this just means MetPy's not trying to map to e.g 'T'--we're still able to map a variable with `axis: T` to `time`, correct?

Reply:
Yes, we are still mapping a coordinate variable with the `axis` attribute of `T` to time. This change here is to the `coordinates` kwarg override in `parse_cf`/`assign_coordinates`, which should now be of the form `{'time': 'coord1', ...}` instead of `{'T': 'coord1', ...}`.